### PR TITLE
Modernize website HTML

### PR DESCRIPTION
**Pull request by smnandre**
A set of minor improvements to modernize the public website
* html5 doctype
* update/clean meta tags
* replace two images with CSS (background gradient / button)
* optimize logo (feather illustrations)
* optimize "fork me" button

Open to any feedback :)

Two questions:
* would it be ok to replace the "fork me on github" rubban with a more "current" github icon ?
* is the google analytics necessary ?

In a following step, i'd like to adapt the CSS to create a responsive version & improve readability on mobile.

### CODE REVIEW

File: theme/static/images/logo.png
Comment (keradus) [inline theme/static/images/logo.png:1]: If you up for it, original logo is here:
https://github.com/PHP-CS-Fixer/logo/blob/master/logo.png

**Comment by keradus** [comment]
> Two questions:
> would it be ok to replace the "fork me on github" rubban with a more "current" github icon ?

i am not sure how new variant is, but likely the answer is "yes"

> is the google analytics necessary ?

I prefer to keep it in place

File: theme/static/images/logo.png
Comment (smnandre) [reply theme/static/images/logo.png:1]: You prefer to use this one ? My "compressed" version looked great to me, but i know logos are a very subjective things so tell me and i'll adapt

**Comment by smnandre** [comment]
> > Two questions:
> > would it be ok to replace the "fork me on github" rubban with a more "current" github icon ?
> 
> i am not sure how new variant is, but likely the answer is "yes"

Github does not encourage anymore the usage of those ribbons (since a few years). 

So just for now i rewrite the one in CSS/vecto, we can then discuss if another option (icon only ? menu ?) would be more adapted ?

| Before | After |
| - | - |
| ribbbefore | ribbbafter |



> I prefer to keep it in place for the website, but if you want to optimize injection of it, go ahead.

Ok, so i put that subject aside for now :)

File: theme/static/images/logo.png
Comment (keradus) [reply theme/static/images/logo.png:1]: I barely see the difference myself (margins only?)
I'm ok with both, i was sharing more in context if you needed original file.

**Comment by keradus** [comment]
I'm OK to have the ribbon discussion aside.
To be honest, I do not mind much which way we do it, but I want to have link to main repo there.

File: theme/layout.html [hunk 1]
```diff
@@ -1,60 +1,46 @@
...
-  </script>
-{%- endmacro %}
+    {%- for cssfile in css_files %}
+      <link rel="stylesheet" href="{{ pathto(cssfile, 1) }}" type="text/css">
+    {%- endfor %}
 
-<html xmlns="http://www.w3.org/1999/xhtml"{% if language is not none %} lang="{{ language }}"{% endif %}>
-  <head>
-    <meta http-equiv="Content-Type" content="text/html; charset={{ encoding }}" />
-    <meta name="robots" content="index, follow, all" />
```
Comment (keradus) [inline theme/layout.html:38]: should we keep this `robots` ?

File: theme/static/css/base.css [hunk 2]
```diff
@@ -203,6 +232,30 @@ a:visited
   padding: 20px 0 0 676px;
 }
 
+.download-button {
+    display: flex;
+    padding: 14px 24px;
+    gap: 16px;
+    font-size: 24px;
+    color: #fff !important;
```
Comment (keradus) [inline theme/static/css/base.css:240]: I was always considering `!important` as hacks ;P
but non-blocking ;)

File: theme/static/css/base.css [hunk 3]
```diff
@@ -203,6 +232,30 @@ a:visited
...
+    background: #FFA210;
+    border-radius: 6px;
+    text-transform: uppercase;
+    font-family: Arial, sans-serif;
+    font-weight: 400;
+    text-decoration: none;
+}
+.download-button svg {
+    display: flex;
+    height: 1.125em;
```
Comment (keradus) [inline theme/static/css/base.css:251]: the button is non-centered, the extra 0.125em going to the bottom of the button.
I would suggest to shift it back to central.

with my poor CSS skills, maybe sth like this?

```suggestion
    height: 1.25em;
    margin-top: -0.125em;
```

**Review by keradus — COMMENTED** [review]
reviewed so far, looking nice. Some minor questions

File: theme/layout.html (hunk 1 above)
Comment (smnandre) [reply theme/layout.html:38]: `all` already includes `Index, follow`...  and is the default value for the meta robots
https://developers.google.com/search/docs/crawling-indexing/robots-meta-tag .. 
So no impact at all there 

But i can guarantee you that the <title> tag fix will have one :)

File: theme/static/css/base.css (hunk 2 above)
Comment (smnandre) [reply theme/static/css/base.css:240]: Haha me too... and i would not have coded that if i had a fresh/blank start 😅

File: theme/static/css/base.css (hunk 3 above)
Comment (smnandre) [reply theme/static/css/base.css:251]: I'll rebase and fix that thx !

Comment (keradus) [reply theme/static/css/base.css:251]: (btw, no need for rebase)

Comment (smnandre) [reply theme/static/css/base.css:251]: I needed to check first, and it appears the icon itself was misaligned.. 

So i've redone it, CSS is smaller.... and svg perfectly aligned :)

**Review by keradus — APPROVED** [review]
thanks!

**Comment by keradus** [comment]
btw, kudos on atomic commits and great commit logs

**Pull request merged**